Cargo.toml

```diff
@@ -49,3 +49,12 @@ assert_matches = "1.4"
 predicates = "1.0"
 rusty-hook = "0.11"
 serial_test = "0.5"
+
+[package.metadata.rpm]
+package = "cloudtruth"
+
+[package.metadata.rpm.cargo]
+buildflags = ["--release"]
```

Comment:
Is it possible to have the cargo-rpm process use the already built binary? Otherwise, you're going to be using to different build flags than all the other releases. Notably, this won't [strip the binary](https://github.com/johnthagen/min-sized-rust#strip-symbols-from-binary) so it will be larger. You can look at the GitHub Action step output to see the pre- and post-strip sizes.

Reply:
Have it use the musl binary?  I'm sure it's possible; I did it this way because you need a rpm capable system to build an rpm, so I built on the centos image separately.  We can do more to make this cleaner, certainly.